src/main/java/ch/ivyteam/ivy/maven/AbstractEngineMojo.java

```diff
@@ -38,7 +38,7 @@ public abstract class AbstractEngineMojo extends AbstractMojo {
    * keep synch with pom.xml &gt; reporting &gt; maven-plugin-plugin &gt;
    * requirements
    */
-  protected static final String MINIMAL_COMPATIBLE_VERSION = "9.3.0";
+  protected static final String MINIMAL_COMPATIBLE_VERSION = "9.4.0";
```

Comment:
didn't know that I've to update this here to. Because I just followed the guide in our confluence page and now I see that the other 50% is documented in this README.md.

![image](https://user-images.githubusercontent.com/7498380/189092575-28fe294f-cb05-4f5a-a403-6c325e01eaee.png)

we should document the release process of an artifact at one place. I would recommend to document that in the REAMDE.md and just pointing to the README.md from the confluence page.

Reply:
Yes, I was confused too about this ... I'll merge into this readme.md, which is currently outdated too